openpower/package/op-test/Config.in

```diff
@@ -0,0 +1,31 @@
+config BR2_PACKAGE_OP_TEST
+        bool "op-test"
+        default y if (BR2_OPENPOWER_PLATFORM)
+        help
+            Include op-test in the Host debug tools tarball.
+
+if BR2_PACKAGE_OP_TEST
+
+choice
+	prompt "op-test version"
+	default BR2_OP_TEST_LATEST_VERSION
+
+config BR2_OP_TEST_LATEST_VERSION
+	bool "Use latest op-test master"
+
+config BR2_OP_TEST_CUSTOM_VERSION
+	bool "Custom op-test version"
+
+endchoice
+
+config BR2_OP_TEST_CUSTOM_VERSION_VALUE
```

Comment:
Not super familiar on how configuration is set. Can you explain how the custom value works. BR2_OP_TEST_CUSTOM_VERSION would be set in witherspoon_defconfig for other branches?